chain/vm/vm.go

```diff
@@ -47,7 +47,7 @@ var (
 
 // ResolveToKeyAddr returns the public key type of address (`BLS`/`SECP256K1`) of an account actor identified by `addr`.
```

Comment:
fix comment

Reply:
and name, as steb says below

Reply:
Changed to ResolveToDeterministicAddr. If we don't feel good with this name, please open an issue to discuss and rename.